Temp adjust

sunjolia opened the pull request:
Temperatures adjusted according to email. Celsius temperatures converted to Fahrenheit. Temperatures are still bimodal, which is to be expected.

sunjolia committed:
Real BBT values calculated based on email. New median values not yet verified

sunjolia committed:
C temps converted to F; Temps still bimodal, expected according to email

sunjolia committed:
Prints median of bimodal temps

sunjolia committed:
Reran 3 over 6 and got slightly lower accuracies

aryamccarthy commented on notebooks/1.0-adm-data-preparation.ipynb:
```diff
@@ -2,7 +2,7 @@
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 9,
```
Jupyter notebooks and Github don't get along well enough. Rerun every file cleanly so that these numbers start at 1 and work their way up.

aryamccarthy replied:
For the record, this is where 95% of your merge issues are coming from.

aryamccarthy commented on notebooks/1.0-adm-data-preparation.ipynb:
```diff
@@ -137,28 +137,43 @@
+    "FAHR = 1\n",
+    "CELS = 2\n",
+    "for i in range (1,100):\n",
+    "    df.ix[df.TEMP_SCALE == FAHR, 'TEMP' + str(i)] =  90 + df.ix[df.TEMP_SCALE == FAHR, 'TEMP' + str(i)]/10\n",
```
Add a Markdown cell above linking to our documentation.

aryamccarthy commented on references/PiacEmail.txt:
```diff
@@ -0,0 +1,15 @@
+Problem Solved for the temperatures (maybe). 
```
Could this get a more informative filename?

aryamccarthy commented on notebooks/1.0-adm-data-preparation.ipynb:
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
You're using Github Desktop, right? There's a button that says "Update from master". Use it. Deal with merge conflicts locally, then push again and there should be no more conflicts, so I'll accept the changes.

sunjolia committed:
Fixed numbering on notebook rows

sunjolia committed:
Tried linking to documentation

In data preparation notebook, in markdown cell above temp reading
alteration, attempted to link to documentation.

aryamccarthy committed:
Merge.

aryamccarthy committed:
Resolve merge conflicts.

aryamccarthy committed:
Passive-aggressive commit message
